src/content/_ContentLoaded.jsx

```diff
@@ -42,6 +47,7 @@ const ContentLoaded = ({ children, type }) => {
         cx(
           theme.className,
           ...(['md', 'mdx'].includes(type) ? ['markdown'] : []),
+          ...(isScrollContainer ? ['scroll'] : []),
```

Comment:
I think you can use a class conditional here. (probably want to double check this)

```suggestion
          { scroll: isScrollContainer },
```

Reply:
That does totally work. Classnames has a pretty flexible api.